ci-operator/step-registry/windows/e2e/operator/test/aws-disconnected/pre/windows-e2e-operator-test-aws-disconnected-pre-chain.yaml

```diff
@@ -0,0 +1,19 @@
+chain:
+  as: windows-e2e-operator-test-aws-disconnected-pre
+  steps:
+    - ref: aws-provision-vpc-disconnected    
+    - chain: aws-provision-bastionhost
+    - ref: ipi-conf
+    - ref: ipi-conf-telemetry
+    - ref: ipi-conf-aws-custom-vpc
+    - ref: ipi-conf-proxy 
+    - ref: ipi-conf-aws
+    - ref: ovn-conf
+    - ref: ovn-conf-hybrid-manifest
+    - chain: ipi-install
+    - ref: aws-provision-tags-for-byo-vpc
+    - ref: enable-qe-catalogsource-disconnected
+    - ref: windows-e2e-operator-test-mirror-images
```

Comment:
Should this be done before the ipi-install chain? Just wondering because I assumed there would be issues mirroring once the cluster comes up in a disconnected state

Reply:
if you take a look at [other jobs ](https://github.com/openshift/release/blob/5d54e4dca965ccee11a469d2321c198c8db30d77/ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/disconnected/provision/cucushift-installer-rehearse-gcp-ipi-disconnected-provision-chain.yaml#L13) they do the mirroring after the install is already completed, since in order to put the images on the cluster's internal registry, there needs to be a cluster to put them on

Though I may be missing the set-sample-operator-disconnected step... I'll add that and re-run the tests.